docs/source/usage.rst

```diff
@@ -1,47 +1,75 @@
 Usage
 =====
 
-Automatic Notebook Versioning
------------------------------
+Data scientists and notebook users may develop locally on their system or save their notebooks to off-site or cloud
+storage. Additionally, they will often create a notebook and then over time make changes and update it. As
+they work, it's helpful to be able to **store versions** of a notebook. When making changes to the content and
+calculations over time, a data scientist using Bookstore can now request different versions from the remote storage,
+such as S3, and **clone** the notebook to their local system.
 
-Every *save* of a notebook creates an *immutable copy* of the notebook on object storage.
+                                             *store*
 
-To simplify implementation, we currently rely on S3 as the object store, using
+Data scientist saves to Local System ------------------> Remote Data Store (i.e. S3) Workspace
+
+                                             *clone*
+
+User requests a notebook to use locally <-------------- Remote Data Store (i.e. S3) Workspace
```

Comment:
Tiny thing, just to be consistent, we should probably use Workspace to mean the semantics associated with workspace. Cloning can be done from any s3 path.

```suggestion
User requests a notebook to use locally <-------------- Remote Data Store (i.e. S3)
```

Reply:
No problem making the change. I want to make sure that I understand correctly how we want to use the word workspace.

My understanding: workspace is a path location to an S3 store where a user stores their notebooks.